Polygon/include/CGAL/Multipolygon_with_holes_2.h

```diff
@@ -78,6 +78,8 @@ class Multipolygon_with_holes_2 {
 
   void add_polygon(const Polygon_2& pgn) { m_polygons.push_back(Polygon_with_holes_2(pgn)); }
 
+  void add_polygon(const Polygon_2&& pgn) { m_polygons.push_back(Polygon_with_holes_2(std::forward(pgn))); }
```

Comment:
I am not sure about `push_back()` vs `emplace_back()`   and about `forward()` vs `move()`.  @sloriot

Reply:
there is no const

Reply:
forward is missing `Polygon_2` as template parameter